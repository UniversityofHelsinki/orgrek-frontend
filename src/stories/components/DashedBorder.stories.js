import React from 'react';
import DashedBorderComponent from '../../components/DashedBorder';
import Box from '@mui/material/Box';

export default {
  component: DashedBorderComponent,
  tags: ['autodocs'],
};

export const DashedBorder = {
  render: () => (
    <Box
      height={150}
      width={150}
      position="relative"
      backgroundColor="background.paper"
      padding={2}
    >
      <DashedBorderComponent />
    </Box>
  ),
};
