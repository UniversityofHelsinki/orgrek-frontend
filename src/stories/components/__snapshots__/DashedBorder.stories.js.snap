// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/components/DashedBorder DashedBorder smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": undefined,
  "name": "stories-components-dashedborder--dashed-border",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
